robots/configBasedRobot.py

```diff
@@ -9,6 +9,7 @@
 import utils.configMapper
 
 class  ConfigBaseCommandRobot(commands2.TimedCommandRobot):
+    balanceing = False
```

Comment:
Is this still neede?

Reply:
It looks like it needs to be moved to `dumboBot.py`

Reply:
I moved the balanceing = False to dumboBot and pushed